tensorflow/python/ops/sets_impl.py

```diff
@@ -167,7 +168,8 @@ def set_intersection(a, b, validate_indices=True):
         ((1, 1, 2), 7),
         ((1, 1, 3), 8),
     ])
-    b = tf.SparseTensor(list(b.keys()), list(b.values()), dense_shape=[2, 2, 4])
+    b = tf.sparse.SparseTensor(list(b.keys()), list(b.values()),
+                                dense_shape=[2, 2, 4])
```

Comment:
```suggestion
                               dense_shape=[2, 2, 4])
```

Please make sure you fix indentation in all examples, not just in those pointed during review

Reply:
Fixed indentation in all examples.